Blish HUD/Blish HUD.csproj

```diff
@@ -201,18 +202,22 @@
     <PackageReference Include="Humanizer.Core.fr" Version="2.6.2" PrivateAssets="all" />
     <PackageReference Include="JsonSubTypes" Version="1.6.0" PrivateAssets="all" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Content.Builder" Version="3.7.0.9" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Extended" Version="[3.7.0]" />
-    <PackageReference Include="MonoGame.Extended.Content.Pipeline" Version="3.7.0" PrivateAssets="all" />
-    <PackageReference Include="MonoGame.Framework.WindowsDX" Version="[3.7.1.189]" />
-    <PackageReference Include="Newtonsoft.Json" Version="[12.0.2]" />
+    <PackageReference Include="MonoGame.Content.Builder.Task" Version="3.8.*" PrivateAssets="all" />
+    <PackageReference Include="MonoGame.Extended" Version="3.8.*" />
+    <PackageReference Include="MonoGame.Extended.Content.Pipeline" Version="3.8.*" PrivateAssets="all" />
+    <PackageReference Include="MonoGame.Framework.WindowsDX" Version="3.8.*" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
I believe we still need the brackets around each version here for when we are a NuGet dependency.